docs/USAGE.md

```diff
@@ -90,7 +90,7 @@ See example with `OTEL_SERVICE` overload.
 
 ### JSON configuration file
 
-By default, if `OTEL_TRACE_CONFIG_FILE` is unset, the application is searching for `datadog.json` in the current working directory (acquired by [`Environment.CurrentDirectory`](https://docs.microsoft.com/en-us/dotnet/api/system.environment.currentdirectory?view=net-5.0)).
+Use environment variable `OTEL_TRACE_CONFIG_FILE` or `web.config` / `app.config` to set configuration file path . This cannot be set via JSON configuration file.
```

Comment:
I do not understand the `This cannot be set via JSON configuration file.` ending. Like it would be strange if anyone would try to do so 😆

Reply:
originally it was somewhere in the docs, so I added just in case 😄